pandas/core/indexes/period.py

```diff
@@ -80,7 +80,7 @@ class PeriodIndex(DatetimeIndexOpsMixin, Int64Index, PeriodDelegateMixin):
 
     Parameters
     ----------
-    data : array-like (1-dimensional), optional
+    data : array-like (1-dimensional) - PeriodArray, optional
```

Comment:
this is not very clear, can you enumerate a bit

1d int np.ndarray or PeriodArray, optional

Reply:
`data : array-like (1-dimensional integer np.ndarray or PeriodArray), optional`, looks good?

Reply:
use 1d

Reply:
Okay, I will get it done after a few hours

Reply:
@jreback I have fixed it, sorry for the delay.